PackageExplorer/PackageChooser/PackageChooserDialog.xaml.cs

```diff
@@ -91,11 +91,7 @@ private void InvokeSearch(string? searchTerm)
 
         private async void Window_Loaded(object sender, RoutedEventArgs e)
         {
-            if (string.IsNullOrEmpty(_pendingSearch))
-            {
-                await Dispatcher.BeginInvoke(new Action(LoadPackages), DispatcherPriority.Background);
```

Comment:
This was a bug because the `PackageSourceBox_OnSelectionChanged` handler is also always called every time the window is shown which caused the packages to load twice (discarding the first load).